src/CalendarDays/CalendarDays.js

```diff
@@ -49,7 +42,12 @@ class CalendarDays extends Component {
       }
 
       squares.push(
-        <CalendarDay day={i} startingDayRecord={startingDayRecord} key={i} />
+        <CalendarDay
+              day={i}
+              month={this.props.month}
```

Comment:
Does a `CalendarDay` object really need a month?

I feel like you can split the components like:
```
<CalendarYear />
```
Which returns a list of 12 `<CalendarMonth />`s,
which returns a list of 31 (or whatever) `<CalendarDay />`s.